api/swagger.yaml

```diff
@@ -668,6 +722,28 @@ paths:
         default:
           $ref: '#/responses/UnknownError'
 
+  /runtime/scheduleScan/start:
```

Comment:
Nothing is being started. start->schedule?

Reply:
scheduledScan

Reply:
removed this path